### PR TITLE
C64 soft80 conio: shave off a few bytes and cycles

### PR DESCRIPTION
**Pull request by Compyx**

### CODE REVIEW

File: libsrc/c64/soft80_cgetc.s [hunk 1]
```diff
@@ -65,20 +75,7 @@ invertcursor:
         ; do not use soft80_putcolor here to make sure the cursor is always
         ; shown using the current textcolor without disturbing the "color voodoo"
         ; in soft80_cputc
```
Comment (greg-king5) [inline libsrc/c64/soft80_cgetc.s:77]: That comment block belongs to the code that was moved to an inlined position.  It, too, should be moved.

**Comment by Compyx** [comment]
Any chance someone will have a look at this? I realize this isn't on the level of compiler fixes, but I have more PR's planned and prefer to do one at a time.

**Comment by oliverschmidt** [comment]
@Compyx: I don't see myself capable of determining if the proposed change does the same as before :-(

@greg-king5: You already reviewed the change. Do you consider this PR as ready to be merged? If so, please merge.

**Comment by greg-king5** [comment]
Sorry, I forgot that I had asked for something to be changed.  It's merged, now.

**Comment by oliverschmidt** [comment]
Thanks for the clarification!

**Comment by Compyx** [comment]
No worries, thanks!

**Pull request merged**